README.md

```diff
@@ -127,58 +128,53 @@ It's possible to package the repo's example application with `electron-builder`.
 
   The configured Linux packaging(s) will be generated in folder `examples/electron/dist`
 
-## Tests
-
-To run tests once, at the root of the project do:
-
-```shell
-yarn test --verbose
-```
+## Using the trace extension
+This section describes how to operate the Theia trace extension to view and analyze traces. The UI and view interactions are prelimiary and subject to revisions in the future.
 
-To keep tests running do:
+![theia-trace-extension](https://raw.githubusercontent.com/ebugden/theia-trace-extension/update-readme/doc/images/theia-trace-extension-0.0.3.png)
 
-```shell
-yarn test --verbose --watch
-```
-
-### Test coverage
+### Open the Trace Viewer
+To open the **Trace Viewer**, select the **Trace Viewer** icon in the left sidebar:
 
-The following command prints a coverage report to the terminal. As of now it covers all typescript files of the project, including those that are not supposed to have tests.
+![Trace Viewer Icon](https://raw.githubusercontent.com/ebugden/theia-trace-extension/update-readme/doc/images/theia-trace-extension-icon.png)
 
-```shell
-yarn test --coverage --collectCoverageFrom='src/**/*.ts'
-```
+#### Add Trace Viewer to Sidebar
+If the **Trace Viewer** icon is not in the left sidebar, select menu **View** from the top-level menu and then select **Trace Viewer** in the list of views. The **Trace Viewer** icon will appear on the left, below the **File Explorer** Icon.
 
-## Using the trace extension
-This section describes how to operate the Theia trace extension to view and analyze traces. The UI, view interactions, and UX are prelimiary and subject to revisions in the future.
+![Add Trace Viewer](https://raw.githubusercontent.com/ebugden/theia-trace-extension/update-readme/doc/images/theia-trace-extension-add-to-sidebar.png)
 
-![theia-trace-extension](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-0.0.2.png)
+![Trace Viewer in Sidebar](https://raw.githubusercontent.com/ebugden/theia-trace-extension/update-readme/doc/images/theia-trace-extension-sidebar.png)
 
 ### Open a trace
-To open a trace in the Theia Trace Extension, use the **File Explorer** in Theia to navigate to the trace directory. Then right-mouse click on the trace and select menu **Open With->Open Trace**.
```

Comment:
Please keep the description to open a trace/trace group from the file explorer. It's a important use case and without the description users won't know about that option. Besides, it's the only way to open a trace that is non-CTF and a single file due to the directory/file dialog limitation.

The description and screenshot needs to be update for the new menu names `Open With-> Trace Viewer`.